bandit/src/get-bandit-tests/dynamo.ts

```diff
@@ -18,4 +18,40 @@ export function queryChannelTests(
 				"#status <> :draft AND isBanditTest = :isBanditTest",
 		})
 		.promise();
+
+	const banner1Tests= docClient
+		.query({
+			TableName: `support-admin-console-channel-tests-${stage.toUpperCase()}`,
+			KeyConditionExpression: "channel = :channel",
+			ExpressionAttributeNames: {
+				"#status": "status",
+			},
+			ExpressionAttributeValues: {
+				":channel": "Banner1",
+				":draft": "Draft",
+				":isBanditTest": true,
+			},
+			FilterExpression:
+				"#status <> :draft AND isBanditTest = :isBanditTest",
+		})
+		.promise();
+	const banner2Tests= docClient
```

Comment:
how about a function instead of this duplication? E.g.
```
const runQuery(channel: string) => docClient.query(...)
return Promise.all([runQuery('Epic'), ...]);
```